package.json

```diff
@@ -9,6 +9,12 @@
     "src"
   ],
   "dependencies": {
-    "neutrino-preset-react": "^2.1.0"
+    "babel-preset-stage-2": "^6.22.0",
+    "deepmerge": "^1.3.2",
+    "neutrino-preset-airbnb-base": "^4.0.1",
+    "neutrino-preset-react": "^4.0.1"
+  },
+  "peerDependencies": {
+    "neutrino": "^4.0.0"
```

Comment:
This one is a `peerDependency`, the actual version used will be in the dependent project.